src/topologyentry.h

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2022-, Intel Corporation
+
+#ifndef TOPOLOGY_ENTRY_HEADER
```

Comment:
we should use "#pragma once" going forward instead of #ifndef include guards

Reply:
Done,